packages/algoliasearch-helper/src/algoliasearch.helper.js

```diff
@@ -1751,15 +1751,18 @@ AlgoliaSearchHelper.prototype._dispatchRecommendResponse = function (
   var results = {};
   Object.keys(idsMap).forEach(function (id) {
     var indices = idsMap[id];
+    var firstResult = content.results[indices[0]];
     if (indices.length === 1) {
-      results[id] = content.results[indices[0]];
+      results[id] = firstResult;
       return;
     }
-    results[id] = sortAndMergeRecommendations(
-      indices.map(function (idx) {
-        return content.results[idx].hits;
-      })
-    );
+    results[id] = Object.assign({}, firstResult, {
```

Comment:
doesn't this break IE11?

Reply:
you can use the merge function from util/functions instead

Reply:
I assumed it was fine as we use `Object.assign` in `RecommendParameters`, means we'd have to change it there too

Reply:
that makes sense, this code is only called if you use recommend indeed